### PR TITLE
Update continuous-integration-and-continuous-delivery.md

### PR DESCRIPTION
**Pull request by vidyavrat**
Updated reference of VSO in this article with VSTS and also modified the URL

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@GuardRex  please review

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ The following diagram from [a Microsoft Patterns and Practices e-book about cont
...
 
-## Visual Studio Online
+## Visual Studio Team Services (VSTS)
 
-One of the things we're working on with Visual Studio Online (VSO), formerly known as Team Foundation Service, is to make it work really well with continuous integration and delivery. Here are some key features of VSO:
+VSTS provided all the features and aspects of application development from planning to deployment. Here are some key features of VSTS:
```
Comment (GuardRex) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md]: * The past tense "provided" should be present tense "provides"
* "all the features and aspects" seems vague and overstated

How about something along the lines of ...

VSTS provides a number of features to assist you with application development from planning to deployment.

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ One of the things we're working on with Visual Studio Online (VSO), formerly kno
 - It supports team room collaboration, which facilitates real-time communication and collaboration for small agile teams.
 - It supports agile project management.
 
-For more information about the continuous integration and delivery features of VSO, see [Visual Studio Lab Management](https://www.visualstudio.com/en-us/explore/lab-management-vs.aspx) and [Visual Studio Release Management](https://www.visualstudio.com/en-us/explore/release-management-vs.aspx). An application monitoring feature, [Application Insights for Visual Studio Online](https://msdn.microsoft.com/en-us/library/dn481095.aspx), is in preview (available to try but not released for production use yet).
+For more information about the continuous integration and delivery features of VSTS, see [https://www.visualstudio.com/team-services/]
```
Comment (GuardRex) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md]: * Link by doc name, which in this case is the VSTS webpage title
* Period at the end of the sentence
* I usually go with "on" over "about" when the object is the definitive source of information. The VSTS doc isn't just "about" the service ... it's **the doc "on" the service**.
```
For more information on the continuous integration and delivery features of VSTS, see [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
```

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md (hunk 1 above)
Comment (vidyavrat) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md]: Thanks @GuardRex I do agree and I should have been more careful about "present tense". "VSTS provides a number of features to assist you with application development from planning to deployment."

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md (hunk 2 above)
Comment (vidyavrat) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md]: Thanks @GuardRex I agree.

**Comment by Rick-Anderson** [comment]
@GuardRex  let me know when you sign off on this.

**Comment by GuardRex** [comment]
@vidyavrat Are you going to provide a commit to address those small issues?

**Comment by vidyavrat** [comment]
https://github.com/aspnet/Docs/pull/2798/commits/2244f7b92284c1d3f12a560c4679fcb4b813b0b9
Kindly review Vidya Vrat Agarwal. Microsoft MVP+1-215-917-2091
www.MyPassionFor.Netwww.Facebook.com/MyPassionFor.Net 

    On Wednesday, February 22, 2017 12:15 PM, Luke Latham <notifications@github.com> wrote:
 

 @vidyavrat Are you going to provide a commit to address those small issues?—

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md [hunk 3]
```diff
@@ -48,9 +48,10 @@ VSTS provided all the features and aspects of application development from plann
 - It supports team room collaboration, which facilitates real-time communication and collaboration for small agile teams.
 - It supports agile project management.
 
-For more information about the continuous integration and delivery features of VSTS, see [https://www.visualstudio.com/team-services/]
 
-If you're looking for a turn-key project management, team collaboration, and source control solution, check out VSO. The service is free for up to 5 users, and you can sign up for it at [https://www.visualstudio.com](https://www.visualstudio.com/). Summary
+For more information on the continuous integration and delivery features of VSTS, see [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
+
+If you're looking for a turn-key project management, team collaboration, and source control solution, check out VSTS. The service is free for up to 5 users, and you can sign up for it at [Visual Studio Team Services](https://www.visualstudio.com/team-services/). Summary
```
Comment (GuardRex) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md]: @vidyavrat Looks like it needs a blank line after the paragraph.

**Comment by vidyavrat** [comment]
Please review again.
 Vidya Vrat Agarwal. Microsoft MVP+1-215-917-2091
www.MyPassionFor.Netwww.Facebook.com/MyPassionFor.Net 

    On Wednesday, February 22, 2017 1:09 PM, Luke Latham <notifications@github.com> wrote:
 

 @GuardRex commented on this pull request.In aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md:>  
-If you're looking for a turn-key project management, team collaboration, and source control solution, check out VSO. The service is free for up to 5 users, and you can sign up for it at [https://www.visualstudio.com](https://www.visualstudio.com/). Summary
+For more information on the continuous integration and delivery features of VSTS, see [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
+
+If you're looking for a turn-key project management, team collaboration, and source control solution, check out VSTS. The service is free for up to 5 users, and you can sign up for it at [Visual Studio Team Services](https://www.visualstudio.com/team-services/). Summary
@vidyavrat Looks like it needs a blank line after the paragraph.—

**Comment by GuardRex** [comment]
@Rick-Anderson :shipit:

**Pull request merged**